src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -780,7 +780,11 @@ private HashSet<string> WriteFiles(ZipArchive package, HashSet<string> filesWith
                         }
                         else
                         {
+#if IS_NET5_OR_LATER
```

Comment:
I thought the framework defines some constants. https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md#what-about-net-10

Reply:
Thanks Nikolche! 
I tried not to use NET5_0, as considering the forwards compatibility.
But it looks like the const only have backwards compatibility.
`In order to make it easier to update code, especially when doing multi-targeting, we should make them additive, so that when targeting net6.0 both NET6_0 and NET5_0 are defined. The same applies to OS bindings.`

Reply:
So in that case, NET5_0 only should do it for us, correct?

Reply:
Yes, you're right! NET5_0 should work in this case, even if we target net6.0 in future.

Reply:
That is true, although I am not sure whether NET5_0 works yet.  @sfoslund is working on that here: https://github.com/dotnet/sdk/pull/11635

Reply:
Tested just now, NET5_0 doesn't work yet. 
Will define it by myself as a temporary workaround.